include/realizations/catchment/Bmi_Adapter.hpp

```diff
@@ -151,28 +152,28 @@ namespace models {
             /** Whether model ``Update`` calls are allowed and handled in some way by the backing model. */
             bool allow_model_exceed_end_time = false;
             /** Path (as a string) to the BMI config file for initializing the backing model (empty if none). */
-            std::string bmi_init_config;
+            string bmi_init_config;
```

Comment:
Wondering if leaving the namespace in the doc strings is a good idea to be clear which namespace these are coming from?  May not be a big deal...